es-vue-base/src/lib-components/EsErrorPage.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+    <div class="my-5 container">
+        <div class="mb-5 mx-auto col-md-7 text-center">
+            <slot name="errorType" />
+            <slot name="errorMsg" />
+        </div>
+
+        <div class="row">
+            <a
+                :class="linkClass"
+                :style="linkStyle"
+                href="mailto:hello@energysage.com">
+                <div class="circle mb-2">
+                    <div
+                        class="m-3 text-orange">
+                        <IconContactUs
+                            :width="iconFill"
+                            :height="iconFill" />
+                    </div>
+                </div>
+
+                <p :class="linkTextClass">Contact Us</p>
+            </a>
+            <a
+                :class="linkClass"
+                :style="linkStyle"
+                href="https://www.energysage.com/solar/">
+                <div class="circle mb-2">
+                    <div
+                        class="m-3 text-orange">
+                        <IconCommunitySolar
+                            :width="iconFill"
+                            :height="iconFill" />
+                    </div>
+                </div>
+                <p :class="linkTextClass">Learn About Solar</p>
+            </a>
+            <a
+                :class="linkClass"
+                :style="linkStyle"
+                href="https://news.energysage.com/">
+                <div class="circle mb-2">
+                    <div
+                        class="m-3 text-orange">
+                        <IconInfoBlogPost
+                            :width="iconFill"
+                            :height="iconFill" />
+                    </div>
+                </div>
+                <p :class="linkTextClass">Informative Blog Posts</p>
+
+            </a>
+            <a
+                :class="linkClass"
+                :style="linkStyle"
+                href="https://www.energysage.com/market-intro/">
+                <div class="circle mb-2">
+                    <div
+                        class="m-3 text-orange">
+                        <IconMarketplace
+                            :width="iconFill"
+                            :height="iconFill" />
+                    </div>
+                </div>
+                <p :class="linkTextClass">EnergySage Marketplace</p>
+            </a>
+        </div>
+    </div>
+</template>
+
+<script lang="js">
+import IconCommunitySolar from '../lib-icons/icon-community-solar.vue';
+import IconContactUs from '../lib-icons/icon-contact-us.vue';
+import IconMarketplace from '../lib-icons/icon-marketplace.vue';
+import IconInfoBlogPost from '../lib-icons/icon-info-blog-post.vue';
+
+export default {
+    name: 'EsErrorPage',
+    components: {
+        IconCommunitySolar,
+        IconContactUs,
+        IconMarketplace,
+        IconInfoBlogPost,
+    },
+    data() {
+        return {
+            linkClass: 'col-6 col-md-3 text-center',
```

Comment:
Generally, I think data() is probably not the place for repeated style classes like this.  It's a great instinct to have as little repeated code as possible, but it's also okay to just repeat bootstrap style classes like in the es-site page.  Or alternatively, computed properties might be a better place to reuse classes than data.

If you want though, you could also see if it works to refactor these links as a `v-for`, and use data to instead keep a list of links+icons to loop through!  It's not necessary but might also be a satisfying option.